view.py

```diff
@@ -0,0 +1,14 @@
+def get_data():
+    
+    return f"{input('Surname: ')} {input('Name: ')} {input('Grade: ')}\n"
+
+def get_number_operation():
+    return input('Choose the category:\n \
+             1-All the Surnames\n \
+             2-All the Names\n \
+             3-All the Grades\n \
+             4-Students of the selected grade\n \
+             5-Entire students table\n \
+             6-Individual student information\n \
+             7-To add new student\n \
+             8-Exit\n ')  
```

Comment:
У вас большой отступ от левого края, сделайте так:
```suggestion
    return input('Choose the category:\n'
                 '1-All the Surnames\n'
                 '2-All the Names\n'
                 '3-All the Grades\n'
                 '4-Students of the selected grade\n'
                 '5-Entire students table\n'
                 '6-Individual student information\n'
                 '7-To add new student\n'
                 '8-Exit\n ')
```